go/genkit/genkit.go

```diff
@@ -292,6 +295,18 @@ func DefineEvaluator(g *Genkit, provider, name string, options *ai.EvaluatorOpti
 	return evaluator
 }
 
+// DefineBatchEvaluator registers the given evaluator function as an action, and
+// returns a [Evaluator] that runs it. This method provide the full
+// [EvaluatorRequest] to the callback function, giving more flexibilty to the
+// user for processing the data, such as batching or parallelization.
+func DefineBatchEvaluator(g *Genkit, provider, name string, options *ai.EvaluatorOptions, eval func(context.Context, *ai.EvaluatorRequest) (*ai.EvaluatorResponse, error)) ai.Evaluator {
+	evaluator, err := ai.DefineBatchEvaluator(g.reg, provider, name, options, eval)
+	if err != nil {
```

Comment:
I just noticed this, we should be returning the error here and in `DefineEvaluator`. We don't need to keep the interface the same as other primitives if it's possible to error during definition.